pkg/cmd/drtprod/scripts/tpcc_init.sh

```diff
@@ -20,12 +20,14 @@ if [ -z "${WORKLOAD_CLUSTER}" ]; then
 fi
 
 # script is responsible for importing the tpcc database for workload
-roachprod ssh "${WORKLOAD_CLUSTER}":1 -- "tee tpcc_init.sh > /dev/null << EOF
+roachprod ssh "${WORKLOAD_CLUSTER}":1 -- "tee tpcc_init.sh > /dev/null << 'EOF'
+TPCC_DB=cct_tpcc
```

Comment:
We need not extract it in an environment variable as it si used in only one place. But, I am fine either ways. Please do test that the generated script works before merging. Thanks

Reply:
Already tested